godtools.xcodeproj/project.pbxproj

```diff
@@ -6290,6 +6291,24 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
+		4503BEFB2821B6E500629692 /* Download And Add Initial Resources and Languages JSON Files */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Download And Add Initial Resources and Languages JSON Files";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "curl -o godtools/Assets/InitialResources/languages.json https://mobile-content-api.cru.org/languages.json\ncurl -o godtools/Assets/InitialResources/resources.json https://mobile-content-api.cru.org/resources?include=latest-translations,attachments.json\n";
```

Comment:
does curl require an extension on the URL? I would say that you shouldn't add the `.json` to the URLs you are downloading.

so use these 2 urls instead:
```
https://mobile-content-api.cru.org/languages
https://mobile-content-api.cru.org/resources?include=latest-translations,attachments
```

Reply:
also, does this phase run for builds targeting the stage environment as well? should there be a way to use the languages/resources from the stage environment instead in that case?

Reply:
Hey Daniel, the extension wasn't required.  Just updated.

The phase will execute against both staging and production schemes.  Let me see if I can differentiate between them.  That would be nice to add.

Reply:
So the build phase executes against any running scheme (GodTools-Staging, GodTools-Production, godtools, GodTools-Release, etc.) I wonder if I can check which scheme it is from build phase...

Reply:
Hey @frett there isn't a way I can differentiate which scheme the build phase is in.  I think it will be fine as long as we run staging with an active internet connection.